packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -184,14 +188,18 @@ abstract class Domain {
       if (_handlers.containsKey(command))
         return _handlers[command](args);
       throw 'command not understood: $name.$command';
-    }).then<Null>((dynamic result) {
+    }).then<dynamic>((dynamic result) {
```

Comment:
`then<void>`?

Reply:
I think we want `dynamic` here, to match what we get from `_handlers[command](args)`.